README.md

```diff
@@ -51,11 +51,6 @@ use a container engine like Podman to pull an OCI container image with the
 appropriate code and libraries to run the AI Model. This can take a long time to setup, but only on the first run.
 ```
 $ ramalama run instructlab/merlinite-7b-lab
-Copying blob 5448ec8c0696 [--------------------------------------] 0.0b / 63.6MiB (skipped: 0.0b = 0.00%)
```

Comment:
We would still see this if on first run, the user ran:

$ ramalama run instructlab/merlinite-7b-lab

and they don't do the pull explicitly